ambry-frontend/src/main/java/com/github/ambry/frontend/AccountAndContainerInjector.java

```diff
@@ -240,7 +239,7 @@ private void injectAccountAndContainerUsingAccountAndContainerHeaders(RestReques
    */
   private void injectAccountAndContainerUsingAccountAndContainerUri(RestRequest restRequest,
       RestRequestMetricsGroup metricsGroup) throws RestServiceException {
-    NamedBlobPath namedBlobPath = RestUtils.parseInput(RestUtils.getRequestPath(restRequest).getOperationOrBlobId(false));
+    NamedBlobPath namedBlobPath = parseNamedBlobPath(RestUtils.getRequestPath(restRequest).getOperationOrBlobId(false));
```

Comment:
minor: Although it's the original code, but I guess maybe we can remove the RestUtils for getRequestPath as well for consistency right?